include/wx/filehistory.h

```diff
@@ -24,6 +24,13 @@ class WXDLLIMPEXP_FWD_CORE wxMenu;
 class WXDLLIMPEXP_FWD_BASE wxConfigBase;
 class WXDLLIMPEXP_FWD_BASE wxFileName;
 
+enum wxFileHistoryMenuLabelStyle
+{
+    wxFH_HIDE_CURRENT_PATH = 0,
```

Comment:
I'm not quite sure if "current" is very clear here. Maybe `wxFH_HIDE_SAME_PATH`? Or `wxFH_SHOW_NON_UNIQUE_PATH`?

Reply:
I've finally used `wxFH_PATH_SHOW_XXX` names for all of them.